www/%team/index.html.spt

```diff
@@ -18,7 +18,9 @@ is_team_owner = not user.ANON and team.owner == user.participant.username
 {% block head %}
     <link rel="payment" type="text/html" title="Give weekly gift on Gratipay"
           href="https://gratipay.com/{{ team.slug }}/" />
+    <meta name="ROBOTS" content="NOINDEX, NOFOLLOW" />
     <meta name="twitter:card" content="summary" />
+    <meta name="og:image" content="{{ website.asset('gratipay.opengraph.png') }}" />
```

Comment:
This is already declared [here](https://github.com/gratipay/gratipay.com/pull/4125/files#diff-54fc7e025b41a7185c3ff65e3111b6f8R28).